packages/app-builder/public/locales/fr/scenarios.json

```
@@ -62,12 +52,198 @@
+  "edit_date.duration_unit_minutes": "minutes",
+  "edit_date.duration_unit_seconds": "secondes",
+  "edit_date.now": "Maintenant",
+  "edit_date.now.description": "L'heure à laquelle le scénario est exécuté",
+  "edit_date.select_a_field": "Sélectionnez un champ d'horodatage...",
+  "edit_date.title": "Créer une date",
+  "edit_fuzzy_match.algorithm.description.ratio": "Compare les textes qui devraient être presque identiques",
+  "edit_fuzzy_match.algorithm.description.token_set_ratio": "Compare les textes qui devraient être presque identiques, mais avec des mots dans un ordre différent",
+  "edit_fuzzy_match.algorithm.label": "Algorithme de similarité",
+  "edit_fuzzy_match.algorithm.ratio": "full string similarity",
+  "edit_fuzzy_match.algorithm.token_set_ratio": "bag of words similarity",
+  "edit_fuzzy_match.description": "Faire correspondre un champ de texte à une liste de valeurs (<DocLink>en savoir plus</DocLink>)",
+  "edit_fuzzy_match.examples.caption": "exemples",
+  "edit_fuzzy_match.examples.left": "texte de gauche",
+  "edit_fuzzy_match.examples.result": "résultat",
+  "edit_fuzzy_match.examples.right": "texte de droite",
+  "edit_fuzzy_match.fuzzy_match": "correspond à",
+  "edit_fuzzy_match.fuzzy_match_any_of": "correspond à un de",
+  "edit_fuzzy_match.level.high": "haut",
+  "edit_fuzzy_match.level.label": "Niveau correspondant",
+  "edit_fuzzy_match.level.medium": "moyen",
+  "edit_fuzzy_match.operands.label": "Texte correspondant",
+  "edit_fuzzy_match.string_similarity": "Similitude de texte",
+  "edit_fuzzy_match.threshold.label": "Seuil de correspondance",
+  "edit_fuzzy_match.title": "Créer une correspondance de texte",
+  "edit_operand.clear_operand": "Clair",
+  "edit_operand.constant.use_data_type.boolean": "Booléen",
+  "edit_operand.constant.use_data_type.number": "Nombre",
+  "edit_operand.constant.use_data_type.string": "Texte",
+  "edit_operand.constant.use_data_type.string[]": "Liste de texte",
+  "edit_operand.constant.use_data_type.number[]": "Liste de nombres",
+  "edit_operand.constant.use_data_type.boolean[]": "Liste de booléens",
+  "edit_operand.data_type.boolean": "Booléen",
+  "edit_operand.data_type.number": "Nombre",
+  "edit_operand.data_type.number[]": "Liste de nombres",
+  "edit_operand.data_type.string": "Texte",
+  "edit_operand.data_type.string[]": "Liste de texte",
+  "edit_operand.data_type.timestamp": "Horodatage",
+  "edit_operand.operator_discovery.from": "Depuis <Path>{{path}}</Path>",
+  "edit_operand.operator_type.enum_one": "Enumération",
+  "edit_operand.operator_type.enum_other": "Enumérations",
+  "edit_operand.operator_type.field_one": "Champ",
+  "edit_operand.operator_type.field_other": "Champs",
+  "edit_operand.operator_type.function_one": "Fonction",
+  "edit_operand.operator_type.function_other": "Fonctions",
+  "edit_operand.operator_type.list_one": "Liste",
+  "edit_operand.operator_type.list_other": "Listes",
+  "edit_operand.placeholder": "Sélectionnez un opérande...",
```

```suggestion
  "edit_operand.placeholder": "Sélectionnez une valeur...",
```
WDYT ? Opérande c'est un peu technique je trouve comme vocabulaire